MBBSEmu/Session/SessionBase.cs

```diff
@@ -222,7 +225,14 @@ public void SendToClient(byte[] dataToSend)
         {
             if (OutputEnabled)
             {
-                SendToClientMethod(dataToSend.Where(shouldSendToClient).ToArray());
+                var dataToSendSpan = new ReadOnlySpan<byte>(dataToSend);
+                var sessionVariables = new Dictionary<string, string>
```

Comment:
Can we set this in the SessionBase constructor so we're not newing it up with every output?